go.mod

```diff
@@ -3,11 +3,10 @@ module github.com/unee-t/apienroll
 go 1.12
 
 require (
-	github.com/apex/log v1.1.0
-	github.com/aws/aws-sdk-go-v2 v0.7.0
-	github.com/go-sql-driver/mysql v1.4.1
-	github.com/gorilla/mux v1.7.1
-	github.com/tj/assert v0.0.0-20171129193455-018094318fb0 // indirect
+	github.com/apex/log v1.1.1
+	github.com/aws/aws-sdk-go-v2 v0.18.0
+	github.com/go-sql-driver/mysql v1.5.0
+	github.com/gorilla/mux v1.7.3
 	github.com/tj/go v1.8.6
-	github.com/unee-t/env v0.0.0-20190328010323-d7125cae867e
+	github.com/unee-t/env v0.2.2
```

Comment:
hmmmm.... https://github.com/unee-t/env/releases doesn't have 0.2.2

I can see you've not tested this.

Reply:
See my comment in the description of this PR:

> We MUST have reviewed, merged and tag released version 0.2.2 of the https://github.com/unee-t/env codebase.
> See PR unee-t/env#2

Reply:
It's a bit confusing that you want a:

1. unee-t/env
2. unee-t-ins/env

Is this really want you want to do?

Reply:
> 
> 
> It's a bit confusing that you want a:
> 
>     1. unee-t/env
> 
>     2. unee-t-ins/env
> 
> 
> Is this really want you want to do?

This is NOT what I want to do!
I want to get rid of `uneet-ins/env` ASAP and use `unee-t/env` instead.
I can NOT do that since `unee-t/env` will only work with one very specific installation of Unee-T as it is today.

Reply:
Ok, I think it's a good idea to use unee-t/env across the board.